spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -145,6 +146,52 @@
     end
   end
 
+  context 'when True_ID response does not contain a decision product status' do
+    let(:true_id_response_success_2) { JSON.parse(LexisNexisFixtures.true_id_response_success_2) }
+    describe 'when a True_ID Decision product is not present in the response' do
+      it 'excludes decision_product_status from logging' do
+        decision_product = get_decision_product(true_id_response_success_2)
+        expect(decision_product).not_to be_nil
```

Comment:
This expectation was confusing because it's the opposite of the spec title and then you change the value later. Is there a reason to check the value here?

Reply:
remove expect line which is unneccesary ... thanks @soniaconnolly